setup.py

```diff
@@ -84,10 +84,7 @@
         "datasets<=2.16",
         "transformers>=4.36,<4.38",
     ],
-    "exporters-executorch": [
-        "executorch>=0.4.0",
-        "transformers>=4.46",
-    ],
+    # "executorch": "optimum-executorch",
```

Comment:
Why is it commented for now?

Reply:
package doesn't exist yet, we could install it from source after https://github.com/huggingface/optimum-executorch/pull/1 is merged